admin/resources/js/components/dashboard/Dashboard.tsx

```diff
@@ -148,9 +149,9 @@ export const Dashboard: React.FC<{
           data-h2-display="b(flex)"
           style={{ flexDirection: "column" }}
         >
-          <Header />
+          <Header baseUrl={BASE_URL} />
```

Comment:
This is going to conflict with my change  in the PR #1378 .
We need to pass ADMIN_APP_DIR instead of BASE_URL. 
Let's co-ordinate while merging in.